packages/nextjs-mf/src/plugins/JsonpChunkLoading.ts

```diff
@@ -0,0 +1,314 @@
+            //   module.id ||
+            //   module.debugId;
+            this.initialModules.add(module);
+            // @ts-ignore
+            this.initialModulesFound.add(module?.options?.shareKey);
+          }
+        };
+
+        compilation.hooks.optimizeChunks.tap(
+          'AddModulesToRuntimeChunkPlugin',
+          (chunks) => {
+            for (const entrypointModule of compilation.entrypoints.values()) {
+              const entrypoint = entrypointModule.getEntrypointChunk();
+              if (entrypoint.name === compiler.options.output.uniqueName)
+                continue;
+
+              const processChunks = (
+                chunks: Set<Chunk>,
+                callback: (modules: Iterable<Module>, chunk: any) => void
+              ): void => {
+                for (const chunk of chunks) {
+                  const modules =
+                    compilation.chunkGraph.getChunkModulesIterableBySourceType(
+                      chunk,
+                      'consume-shared'
+                    );
+                  if (modules) callback(modules, chunk);
+                }
+              };
+
+              processChunks(entrypoint.getAllInitialChunks(), addModules);
+              processChunks(
+                entrypoint.getAllAsyncChunks(),
+                (modules, chunk) => {
+                  for (const m of modules) {
+                    if (
+                      // @ts-ignore
+                      m?.options &&
+                      // @ts-ignore
+                      !this.initialModulesFound.has(m?.options?.shareKey) &&
+                      // @ts-ignore
+                      m?.options?.shareKey === 'next/router'
+                    ) {
+                      addModules([m], chunk);
+                      // @ts-ignore
+                      this.initialModulesFound.add(m?.options?.shareKey);
+                    }
+                  }
+                }
+              );
+            }
+          }
+        );
+
+        compilation.hooks.runtimeModule.tap(
+          'CustomWebpackPlugin',
+          (runtimeModule: RuntimeModule, chunk: any) => {
+            if (
+              runtimeModule.constructor.name ===
+                'JsonpChunkLoadingRuntimeModule' &&
+              chunk.name === 'webpack'
+            ) {
+              const originalSource = runtimeModule.getGeneratedCode();
+              // @ts-ignore
+              const modifiedSource = new ConcatSource(
+                originalSource,
+                '\n',
+                '// Custom code here\n',
+                getCustomJsonpCode(
+                  //@ts-ignore
+                  compiler.options.output.uniqueName,
+                  compiler.webpack.RuntimeGlobals,
+                  // @ts-ignore
+                  this.initialModules
+                )
+              );
+              runtimeModule.getGeneratedCode = () => modifiedSource.source();
```

Comment:
Replace the code generation cal in the module to my updated one. I don't want to regenerate its source, just add to the bottom of it